### PR TITLE
Improve validation language around timestampWrites

### PR DESCRIPTION
**Pull request by kainino0x**
- Deduplicates effectively identical between compute and render pass descriptor validation
- Actually links to this validation from beginComputePass, instead of not
- Consolidate validation that was unnecessarily split between begin*Pass and the descriptors' valid usage blocks
- Removes an unnecessary `Issue`
- Fixes an unrelated typo

(Note, this is not related to #3808.)

### CODE REVIEW

File: spec/index.bs [hunk 1]
```diff
@@ -12606,6 +12579,22 @@ Issue: Write normative text about timestamp value resets.
...
+<div algorithm class=validusage>
+    <dfn abstract-op>Validate timestampWrites</dfn>({{GPUDevice}} |device|,
+    <code>({{GPUComputePassTimestampWrites}} or {{GPURenderPassTimestampWrites}})</code> |timestampWrites|)
+
+    Return `true` if the following requirements are met, and `false` if not.
+
+    - If |timestampWrites| is not [=list/is empty|empty=],
+        {{GPUFeatureName/"timestamp-query"}} must be [=enabled for=] |device|.
+    - No two entries in |timestampWrites| may have the same `location`.
+    - No two entries in |timestampWrites| may have the same `querySet` and `queryIndex` pair.
```
Comment (haoxli) [inline spec/index.bs:12591]: The same `querySet` and `queryIndex` pair is allowed on compute pass, this should only be validated on render pass.

Comment (kainino0x) [reply spec/index.bs:12591]: Well spotted! Thanks for reviewing, will fix soon.

Comment (kainino0x) [reply spec/index.bs]: FWIW I think #3808 _will_ make this obsolete, but the goal here with this PR is not to change anything.
Moved this validation to the render pass only.

**Review by haoxli — APPROVED** [review]
LGTM

**Pull request merged**